.github/workflows/osisoft-connector-release-build.yml

```diff
@@ -0,0 +1,178 @@
+
+      - name: Generate Javadocs for Java Files in /src and /libs (Ubuntu)
+        run: javadoc -private -splitindex -use -author -version -d $TMP_JAVADOC_FOLDER -classpath ./$TMP_CP_FOLDER/$ETK_LOCAL_FILE_NAME $(find $SRC_FOLDER -name "*.java")
+
+      - name: Build Ewon Jar in Ant (Ubuntu)
+        run: ant "buildjar" -noinput -buildfile build.xml
+
+      - name: Minify Javascript
+        uses: nizarmah/auto-minify@master
+        with:
+          directory: 'web/js'
+          output: 'mini_js'
+
+      - name: Minify CSS
+        uses: nizarmah/auto-minify@master
+        with:
+          directory: 'web/css'
+          output: 'mini_css'
+
+      - name: Parse Ant Build File for Project Name (Ubuntu)
+        run: |
+          export PROJECTNAME=$(sed -n 's/<property name=\"ProjectName\"\ value=\"\(.*\)\"\/>/\1/p' build.xml | tr -d '[:space:]')
+          echo "PROJECT_NAME=$PROJECTNAME" >> $GITHUB_ENV
+
+      - name: Copy min js to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "mini_js/*.min.js"
+          target: "osisoft_connector/js/"
+
+      - name: Copy config to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "config/*"
+          target: "osisoft_connector/"
+
+      - name: Copy min css to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "mini_css/*.min.css"
+          target: "osisoft_connector/css/"
+
+      - name: Copy min external js to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "web/js/external/."
+          target: "osisoft_connector/js/"
+          flags: '-r'
+
+      - name: Copy bootstrap min js to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "web/bootstrap/js/*min.js"
+          target: "osisoft_connector/js/"
+
+      - name: Copy changelog to root directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "docs/CHANGELOG.md"
+          target: "./"
+
+      - name: Copy bootstrap css to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "web/bootstrap/css/*.min.css"
+          target: "osisoft_connector/css/"
+
+      - name: Copy scripts to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "scripts/*"
+          target: "osisoft_connector/"
+
+      - name: Copy jar file to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "build/*.jar"
+          target: "osisoft_connector/"
+
+
+      - name: Copy html file to release directory
+        uses: canastro/copy-file-action@master
+        with:
+          source: "web/html/*.html"
+          target: "osisoft_connector/"
```

Comment:
Any reason you are not just using bash here in one task? It would probably clean up this chunk of the action.

Reply:
This also just seems like a lot of overhead to encapsulate ```cp``` in docker

Reply:
This wasn't working for me when calling bash commands from a task, so I moved all relevant commands into a single script which is called to set up the release directory.